elfo-core/src/errors.rs

```diff
@@ -1,14 +1,40 @@
+use std::fmt::Display;
+
 use derive_more::{Display, Error};
 
 #[derive(Debug, Display, Error)]
 #[non_exhaustive]
 pub enum StartError {
-    /// Configs must be valid at the start-up.
-    #[display(fmt = "invalid config")]
-    InvalidConfig,
+    #[display(fmt = "error starting entrypoints: {}", _0)]
+    EntrypointErrors(#[error(not(source))] EntrypointErrors),
```

Comment:
Double `Error` in `StartError::EntrypointErrors`. What about `EntrypointFailed` or simply `Entrypoint`?